Docker packaging needs updated for #239

**What happened**:
The Dockerfile does not package up `plugin.json`.  I'll submit a PR to fix this shortly.

**What you expected to happen**:
The `/render/version` API to work; added in #239.

**How to reproduce it (as minimally and precisely as possible)**:

Start a container
```
$ docker run -d --name test grafana/grafana-image-renderer:3.0.0
```

Exec into it:
```
$ docker exec -it test sh
```

Install curl and try to hit the version API endpoint:
```
# apk add curl
# curl localhost:8081/render/version
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Internal Server Error</pre>
</body>
</html>
```

Check the logs from the renderer:

```
$ docker logs test
{"level":"info","message":"HTTP Server started, listening at http://localhost:8081"}
{"message":"::ffff:127.0.0.1 - - [08/Jun/2021:13:56:09 +0000] \"GET /renderer/version HTTP/1.1\" 404 155 \"-\" \"curl/7.77.0\"\n","level":"error"}
{"url":"/render/version","stack":"Error: Cannot find module '../../plugin.json'\nRequire stack:\n- /usr/src/app/build/service/http-server.js\n- /usr/src/app/build/app.js\n    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:885:15)\n    at Function.Module._load (internal/modules/cjs/loader.js:730:27)\n    at Module.require (internal/modules/cjs/loader.js:957:19)\n    at require (internal/modules/cjs/helpers.js:88:18)\n    at /usr/src/app/build/service/http-server.js:131:36\n    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)\n    at next (/usr/src/app/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/usr/src/app/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)\n    at /usr/src/app/node_modules/express/lib/router/index.js:281:22","level":"error","message":"Request failed"}
{"message":"::ffff:127.0.0.1 - - [08/Jun/2021:13:56:23 +0000] \"GET /render/version HTTP/1.1\" 500 148 \"-\" \"curl/7.77.0\"\n","level":"error"}
TypeError: Cannot read property 'statusCode' of undefined
    at /usr/src/app/build/service/http-server.js:143:46
    at Layer.handle_error (/usr/src/app/node_modules/express/lib/router/layer.js:71:5)
    at trim_prefix (/usr/src/app/node_modules/express/lib/router/index.js:315:13)
    at /usr/src/app/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/usr/src/app/node_modules/express/lib/router/index.js:335:12)
    at next (/usr/src/app/node_modules/express/lib/router/index.js:275:10)
    at next (/usr/src/app/node_modules/express/lib/router/route.js:127:14)
    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:97:5)
    at next (/usr/src/app/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/usr/src/app/node_modules/express/lib/router/route.js:112:3)
```

**Anything else we need to know?**:

**Environment**:
- Grafana Image Renderer version: 3.0.0
- Grafana version: 8.0.0
- Installed plugin or remote renderer service: remote
- OS Grafana Image Renderer is installed on: n/a
- User OS & Browser:
- Others:

--- a/Dockerfile
+++ b/Dockerfile
@@ -32,6 +32,7 @@ COPY --from=build /usr/src/app/node_modules node_modules
 COPY --from=build /usr/src/app/build build
 COPY --from=build /usr/src/app/proto proto
 COPY --from=build /usr/src/app/default.json config.json
+COPY --from=build /usr/src/app/plugin.json plugin.json
 
 EXPOSE 8081
 

--- a/debian.Dockerfile
+++ b/debian.Dockerfile
@@ -34,6 +34,7 @@ COPY --from=build /usr/src/app/node_modules node_modules
 COPY --from=build /usr/src/app/build build
 COPY --from=build /usr/src/app/proto proto
 COPY --from=build /usr/src/app/default.json config.json
+COPY --from=build /usr/src/app/plugin.json plugin.json
 
 EXPOSE 8081
 

